### ISSUE TITLE
Change plugin_manager instance to prevent confusion with module (Trac #74)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#78]
The app_context module has a module-level variable named plugin_manager that is an instance of the PluginManager class. This is a bit confusing since many modules import the plugin_manager module and also the plugin_manager instance; it is difficult to distinguish between them. Instead, we can replace the plug_manager instance with a method called get_plugin_manager.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/74
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:43", 
    "description": "The app_context module has a module-level variable named plugin_manager that is an instance of the PluginManager class. This is a bit confusing since many modules import the plugin_manager module and also the plugin_manager instance; it is difficult to distinguish between them. Instead, we can replace the plug_manager instance with a method called get_plugin_manager.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763583719301", 
    "component": "microdrop", 
    "summary": "Change plugin_manager instance to prevent confusion with module", 
    "priority": "minor", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-03-15T04:31:49", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120315T04:40:21: ryan changed description from "The app_context module has a module-level variable named plugin_manager that is an instance of the PluginManager class. This is a bit confusing since many modules import the plugin_manager module and also the plugin_manager instance; it is easy to distinguish them. Instead, we can replace the plug_manager instance with a method called get_plugin_manager." to "The app_context module has a module-level variable named plugin_manager that is an instance of the PluginManager class. This is a bit confusing since many modules import the plugin_manager module and also the plugin_manager instance; it is difficult to distinguish between them. Instead, we can replace the plug_manager instance with a method called get_plugin_manager."

**Comment by cfobel** [comment]
Trac update at 20120315T05:16:59: ryan changed status from "new" to "assigned"

**Comment by cfobel** [comment]
Trac update at 20120327T20:54:28: 
* ryan@fobel.net commented:

> commit 39be8cfb213f4c38c294ebc336a45e77e39f345b
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Tue Mar 27 16:51:38 2012 -0400
> 
>     Remove PluginManager class and move methods to the module
>     
>     Fixes #78.


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "assigned" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:43: ryan changed component from "plugins" to "microdrop"
